deploy/test-environments/fleet_api/src/api/common_api.py

```diff
@@ -256,6 +302,48 @@ def get_package_version(
         return None
 
 
+def get_package(
+    cfg: Munch,
+    package_name: str = "cloud_security_posture",
+    is_full: bool = True,
+    prerelease: bool = False,
+) -> Dict[str, Any]:
+    """
+    Retrieve package information from the Elastic Fleet Server API.
+
+    Args:
+        cfg (Munch): Configuration data.
+        package_name (str, optional): The name of the package to retrieve.
+                                      Default is "cloud_security_posture".
+        is_full (bool, optional): Whether to retrieve full package information. Default is True.
+        prerelease (bool, optional): Whether to include prerelease versions. Default is False.
+
+    Returns:
+        Dict[str, Any]: A dictionary containing the package information
+                        or an empty dictionary if the API call fails.
+    """
+    url = f"{cfg.kibana_url}/api/fleet/epm/packages/{package_name}"
+
+    request_params = {
+        "full": is_full,
+        "prerelease": prerelease,
+    }
+
+    try:
+        response = perform_api_call(
+            method="GET",
+            url=url,
+            auth=cfg.auth,
+            params={"params": request_params},
+        )
+        return response.get("response", {})
+    except APICallException as api_ex:
+        logger.error(
+            f"API call failed, status code {api_ex.status_code}. Response: {api_ex.response_text}",
+        )
+        return {}
```

Comment:
why return an empty dict on an error? what will that cause?

Reply:
After our discussion, I replaced the `return` statement with `raise` to halt the execution of the entire script when an exception occurs